src/evm-block-extractor/src/config.rs

```diff
@@ -108,13 +109,18 @@ impl Database {
                 log::info!("- project-id: {}", project_id);
                 log::info!("- dataset-id: {}", dataset_id);
 
-                if !dataset_id.contains(TESTNET_PREFIX) && !dataset_id.contains(MAINNET_PREFIX) {
+                if ![TESTNET_PREFIX, MAINNET_PREFIX, DEVNET_PREFIX]
+                    .iter()
+                    .any(|&prefix| dataset_id.contains(prefix))
+                {
                     return Err(anyhow::anyhow!(
-                        "Invalid dataset ID. The dataset ID must be prefixed with {} or {}",
+                        "Invalid dataset ID. The dataset ID must be prefixed with {}, {}, or {}",
                         TESTNET_PREFIX,
-                        MAINNET_PREFIX
+                        MAINNET_PREFIX,
```

Comment:
What is the reason to restrict this value to a fixed set of values?

Reply:
This is mainly used to avoid confusion in the bigquery table but since we are using in three different projects, I guess we don't need to restrict it.